src/kimmdy/coordinates.py

```diff
@@ -292,69 +422,98 @@ def merge_top_moleculetypes_slow_growth(
                     c2=parameterizedB.c0,
                     c3=parameterizedB.c1,
                 )
+
+            # bond only exists in A -> vanish bond
             elif parameterizedA:
                 atomtypes = [molA.atoms[atom_id].type for atom_id in key]
                 break_bind_atoms[key[0]] = atomtypes[0]
                 break_bind_atoms[key[1]] = atomtypes[1]
-                # use combination rule type 2 (typically used by amber force fields)
-                sigmas = [ff.atomtypes[at].sigma for at in atomtypes]
-                sigmaij = 0.5 * (float(sigmas[0]) + float(sigmas[1]))
-                epsilons = [ff.atomtypes[at].epsilon for at in atomtypes]
-                epsilonij = np.sqrt(float(epsilons[0]) * float(epsilons[1]))
-
-                # morse well steepness
-                assert parameterizedA.c1 is not None, f"parameterizedA.c1 is not set"
-                beta = np.sqrt(
-                    float(parameterizedA.c1) / (2 * hyperparameters["morse_well_depth"])
-                )
+                sigmaij, epsilonij = get_LJ_parameters(*key)
 
                 molB.bonds[key] = Bond(
                     *key,
-                    funct="3",
-                    c0=parameterizedA.c0,
-                    c1=f"{hyperparameters['morse_well_depth']:5.3f}",
-                    c2=f"{beta:5.3f}",
-                    c3=f"{sigmaij*1.12:7.5f}",  # sigmaij* 1.12 = LJ minimum
-                    c4=f"{epsilonij:7.5f}",  # well depth is epsilonij
-                    c5=f"{beta:5.3f}",
+                    funct=FFFUNC["morse_bond"],
+                    c0=parameterizedA.c0,  # b
+                    c1=f"{hyperparameters['morse_well_depth']:.5f}",  # D
+                    c2=f"{hyperparameters['beta']:.5f}",  # beta
+                    c3=f"{sigmaij*1.12:.5f}",  # sigmaij* 1.12 = LJ minimum
+                    c4=f"{0.0:.5f}",  # well depth -> zero
+                    c5=f"{0.0:.5f}",
                 )
 
                 # update bound_to
                 atompair = [molB.atoms[key[0]], molB.atoms[key[1]]]
                 atompair[0].bound_to_nrs.append(atompair[1].nr)
                 atompair[1].bound_to_nrs.append(atompair[0].nr)
 
+                # Find all neighbors of the bond
+                neighbor_sides = []
+                for idx in key:
+                    bonds = list(filter(lambda b: idx in b, keysB))
+                    neighbor_set = set()
+                    for b in bonds:
+                        neighbor_set.add(b[0])
+                        neighbor_set.add(b[1])
+                    neighbor_sides.append(neighbor_set)
+                neighbor_set.discard(idx)  # avoid double counting central bond
```

Comment:
Just spotted this. This is intentionally making use of the fact that the iterator in a loop stays bound to the last iteration of the loop, right?

For extra pickiness: I had started cleaning up the use of `idx` in favor of either `id` or `ix` to clearly separate 1-based string ids from 0-based integer indices.

Reply:
yes it does. I think it's not required anymore, as the pairs are deduplicated later anyway, but it is working as inteded ;)